src/common/errors/errors.go

```diff
@@ -15,6 +15,7 @@
  *
  *******************************************************************************/
 
+// Package errors defines error cases of edge-orchestration.
 package errors
```

Comment:
How about to change package name?
because this package name can be confused with golang default package "errors".

Reply:
I agree with the suggestion that there will be confusion due to the package name. After the discussion, I will reflect it as another PR.